src/scripts/towns/Town.ts

```diff
@@ -371,9 +371,6 @@ TownList['Cerulean Cave'] = new DungeonTown(
 const NewBarkTownShop = new TownShop([
     ItemList['Pokeball'],
 ]);
-const CherrygroveCityShop = new TownShop([
-    ItemList['Greatball'],
```

Comment:
Are these still available elsewhere in the region?

Reply:
Nvm, found em 👍

Reply:
Yes they are in Ecruteak.